app/assets/javascripts/editor.js

```diff
@@ -23,131 +29,119 @@ $E = {
         return code;
       }
     });
-  },
-  setState: function(textarea = 'text-input', preview = 'comment-preview-main', title = 'title') {
-    $E.title = $('#' + title + 'title'); // not sure why this exists? seems like $E.title is always #title
-    $E.textarea = $('#' + textarea);
+  }
+  setState(commentFormID) {
+    this.commentFormID = commentFormID;
+    this.textarea = $("#text-input-" + commentFormID);
     $E.textarea.bind('input propertychange', $E.save);
```

Comment:
I changed most instances of `$E.method` to `this.method` except for this one here. I toyed around with this, but it opens up a can of worms and is out of scope of this PR. Will go back to this when I refactor save and recover functions.

Reply:
Just to explain this a little further, it's because when `$E.textarea` is changed to `this.textarea` it brings up JavaScript `this` weirdness. The eventListener for `input propertychange` (which I'm not sure is a valid event type? I tried Googling it) gets attached to the wrong `this`.